pkg/apb/types.go

```diff
@@ -32,20 +32,34 @@ type Parameters map[string]interface{}
 // SpecManifest - Spec ID to Spec manifest
 type SpecManifest map[string]*Spec
 
+type nilableNumber float64
```

Comment:
I think you should make this exportable so that it can be used when creating a ParameterDescriptor outside of the APB package.